app/jobs/repository_audit_job.rb

```diff
@@ -0,0 +1,10 @@
+class RepositoryAuditJob < ApplicationJob
+  def perform(user: User.system_user, audit: Audit.new(user: user))
```

Comment:
Since we don't need to pass the audit, we can make it in the body of the method now.

I *think* the serialization would also choke on the system user with `perform_later`, so we will have to try `RepositoryAuditJob.perform_now` to be sure `rails runner` can run the job with a transient instance.